src/atoms/View/__tests__/View.web.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { renderWithTheme } from '../../../_helpers/testing';
+
+import View from '../index';
+
+describe('Renders with text ', () => {
+  it('snapshot testing', () => {
+    const { container } = renderWithTheme(<View>{'simple div'}</View>);
+    expect(container).toMatchSnapshot();
+  });
+});
```

Comment:
```suggestion
describe('<View />', () => {
  describe('as', () => {
    it('renders View as div', () => {
      const { container } = renderWithTheme(
        <View as="div">
          <Text>{'simple div'}</Text>
        </View>,
      );
      expect(container).toMatchSnapshot();
    });
    it('renders View as span', () => {
      const { container } = renderWithTheme(
        <View as="span">
          <Text>{'simple div'}</Text>
        </View>,
      );
      expect(container).toMatchSnapshot();
    });
    it('renders View as p', () => {
      const { container } = renderWithTheme(
        <View as="p">
          <Text>{'simple div'}</Text>
        </View>,
      );
      expect(container).toMatchSnapshot();
    });
  });
});
```

This is how we try to write out tests. I'll send more details on slack. For now you can just use this suggestion for the test.